lib/async/redis/methods/hashes.rb

```diff
@@ -0,0 +1,80 @@
+# Copyright, 2018, by Samuel G. D. Williams. <http://www.codeotaku.com>
+# Copyright, 2018, by Huba Nagy.
```

Comment:
Please put your own copyright here.

Reply:
@ioquatix, I just append my name underneath Huba?

Reply:
@ioquatix just tell me if I need to modify it. Just appended my name underneath Huba

Reply:
Since you created the file feel free to delete me and Huba.

The point of copyright per file is to have a objective source of truth for who actually worked on the file and when.

I know this is provided by git, but you have to assume these files are not always versioned (e.g. when distributed as part of a gem). If, for some reason, we ever needed to relicense the code, we'd need an objective source of truth for who was responsible for what code. So, it's nice to have a record.

Reply:
Done!